content/software-security/cves/cve-why-care.md

```diff
@@ -0,0 +1,58 @@
+
+Vulnerability *triage* is an important step developers must take to work toward reducing risks posed by vulnerabilities in their software. Using [*vulnerability scanners*](https://owasp.org/www-community/Vulnerability_Scanning_Tools) can give developers information about the number, type, and severity of CVEs present in their work. With this data, developers can prioritize critical CVEs first, therefore ensuring that major security concerns are addressed first.
+
+
+## Federal Regulations
+
+With notable software supply chain security attacks occurring in recent years (such as the [SolarWinds attack in 2020](/software-security/glossary/#solarwinds-hack)), the U.S. federal government has increased efforts to improve U.S. cybersecurity. These initiatives aim to strengthen software supply chain security by promoting safe development habits, such as frequent vulnerability scanning.
+
+
+### FedRAMP
+
+The [Federal Risk and Authorization Management Program](https://www.fedramp.gov/program-basics/) (FedRAMP) is a security framework that must be adopted by cloud service providers (CSPs) before they service U.S. federal government agencies. The framework aims to promote the use of cloud services across agencies by standardizing security authorization practices. Based on the service being offered, FedRAMP sorts cloud services into low, moderate, and high impact levels, with increased security expectations for each level.
+
+In order to achieve FedRAMP authorization, certain requirements need to be met as follows: 
+
+- Container images used by CSPs must be *hardened* according to benchmarks laid out in the [NIST SP 800-70](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-70r4.pdf). Chainguard Images offers [hardened, minimal base images](https://www.chainguard.dev/chainguard-images) designed to help meet FedRAMP compliance requirements.
+
+- Vulnerability scanners are expected to report information about discovered vulnerabilities, such as its CVE ID and CVSSv3 score. Additional configuration requirements can be found in the [FedRAMP Vulnerability Scanning Requirements outline](https://www.fedramp.gov/assets/resources/documents/CSP_Vulnerability_Scanning_Requirements.pdf).
+
+Meeting FedRAMP container image and vulnerability scanning requirements allows your organization to expand and reinforce its offerings as a CSP. To learn how Chainguard Images can help you meet FedRAMP container image requirements, check out [this blog post](https://www.chainguard.dev/unchained/fortify-comply-and-conquer-fedramp-with-chainguard-images).
```

Comment:
I would use the title of the blog post as that helps for SEO. You can also use the royal "we" here if you would like. I think the benefit of that is that there is not a gotcha about us being a totally objective source.